.github/workflows/test_e2e.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - name: Run Services with Docker
         working-directory: ./services
         run: |
-          docker --log-level 'warn' compose -f docker-compose.yml up -d --quiet-pull
+          docker --log-level 'warn' compose -f docker-compose.yml up -d --quiet-pull --wait --wait-timeout 300
```

Comment:
Is this a recent addition to Docker to "wait untill all healthchecks are ready"?? Awesome. :)

Reply:
Yes, at least that's what I'm interpreting from: https://stackoverflow.com/a/76422970
But, now the struggle is within the healthcheck itself to get it to return 'Healthy'